### PR TITLE
chore(build): update libs, enable standalone build

### PR DESCRIPTION
**Pull request by anotherchrisberry**
All I wanted to do was put a little padding at the bottom of the report details, and a little inset box shadow to hint that there's more stuff to be scrolled into the view:
Screen Shot 2020-07-02 at 5 34 53 PM

The repository includes a simple standalone version of Deck, but it's been long neglected, and wouldn't run. So I started upgrading libraries to make it compatible with the current versions of `@spinnaker/core` and `@spinnaker/amazon`, and that's what I did today and that was a whole day so I'm done with this now.

The app can actually be started again, and the library build works with our custom build of Deck at Netflix. The current version doesn't after an import change from [this PR](https://github.com/spinnaker/deck-kayenta/pull/500). Not a big deal! This stuff is all very confusing. @fieldju: it's worth pulling this down and confirming the library build works with your custom build.

Most of the webpack config changes were hacked in from OSS Deck and a little bit from our internal build.

**Update**
There are...more changes. I meant to add them as a separate commit but accidentally amended my original commit, so here is the summary of the additional changes:
* using eslint instead of tslint, and using the same configuration as @spinnaker/deck via its eslint plugin.
* using a different tool (but in the same family) for code coverage reporting

I think awesome-typescript-loader or happypack or something was masking some of the compilation errors, but ts-loader does not swallow them, which started the build issues.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'jquery';
```
Comment (caseyhebebrand) [inline src/app.ts:1]: Curious where this uses `jquery`?

Comment (anotherchrisberry) [reply src/app.ts:1]: This is an old-timey angular thing. AngularJS would use its own "lite" version of jQuery by default, but if you need regular jQuery (and there is some old-timey code in Deck that does) in your Angular modules, you have to load jQuery before Angular.

**Comment by caseyhebebrand** [comment]
@anotherchrisberry  For some reason `ts-loader` is unable to transpile when running the unit tests. This is what is causing the build to break, but this does not happen when running the unit tests using `yarn test`, only `buildViaTravis`. 

Is there anyway we can extrapolate the fix for the `Tooltip` change related to [this PR](https://github.com/spinnaker/deck-kayenta/pull/500/files) since this is a blocker? @fieldju - wrt that Tooltip PR, we have the same error that you saw before the PR. When I revert your PR everything works as expected.

**Comment by anotherchrisberry** [comment]
Well that was fun!

**Pull request merged**